tests/define.test.js

```diff
@@ -243,6 +262,27 @@ describe('Plugin for define blocks', () => {
     `);
   });
 
+  it('accounts for variable name conflicts when checking the result of `define`', () => {
+    expect(`
+      var amdDefineResult = 'for some reason I have this variable declared already';
+      define(function(require, exports, module) {
+        var stuff = require('hi');
+        exports.hey = stuff.boi;
+      });
+    `).toBeTransformedTo(
+      `var amdDefineResult = 'for some reason I have this variable declared already';` +
+        checkAmdDefineResult(
+          `
+            (function(require, exports, module) {
+              var stuff = require('hi');
+              exports.hey = stuff.boi;
+            })(require, exports, module)
+          `,
+          `_${AMD_DEFINE_RESULT}`
```

Comment:
This test depends on how babel internally generates unique variable names, which isn't great, but can't really think of another way to test this other than doing some funky regex stuff that just doesn't seem worth it.